css/style.css

```diff
@@ -403,66 +403,72 @@ hr {
 
 #footer {
     padding: 25px 0px 40px 0px;
-    background-image: linear-gradient(130deg, #05272e 40%, #183a36 100%);
+     background-image: linear-gradient(130deg, #05272e 40%, #183a36 100%); 
     /* background-image:linear-gradient(130deg, #ffa159 10%, #ff39b4 51%, #ffa159 100%);    */
 }
 
 .list-group a{
     color: white;
 }
-.subscribe-box {
-    background-image: linear-gradient(130deg, #05272e 40%, #183a36 100%);
-    box-shadow: 2px 2px 3px 5px  #298b80 ,
-    2px 2px 3px 5px black;
-   
-}
+
```

Comment:
any reason to remove the `.subscribe-box` styles?

Reply:
It uses the single background-color property in the contact box that looking wired  so I thought  to remove that